index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<header id="header">
+  <link rel="stylesheet" href="styles.css">
+  <img id="header-img" src="">Logo</img></br>
+  <nav id="nav-bar">
+    
+      <a href="#features" class="nav-link">Features</a>&nbsp;&nbsp;&nbsp;
+      <a href="#video" class="nav-link"> How it works</a>&nbsp;&nbsp;&nbsp;
+      <a href="#pricing" class="nav-link">Pricing</a>
```

Comment:
Kindly enclose the contents of the nav bar in a ul tag. Then, make each item a li tag. This is in line with HTML5 best practices.

Reply:
- [OPTIONAL] Kindly state the reason for your requests. And if possible point students to any resources that could be of help to the students.